### PR TITLE
Add metrics command

### PR DESCRIPTION
**Pull request by edangelion**
Add metrics command for fetching the metric by `/api/v0/hosts/<hostId>/metrics` and `/api/v0/services/<serviceName>/metrics`

### CODE REVIEW

File: commands.go [hunk 1]
```diff
@@ -464,6 +485,38 @@ func split(ids []string, count int) [][]string {
...
 }
 
+func doMetrics(c *cli.Context) error {
+	optHostID := c.String("host")
+	optService := c.String("service")
+	optMetricName := c.String("name")
+
+	from := c.Int64("from")
+	to := c.Int64("to")
+	if from == 0 && to == 0 {
```
Comment (astj) [inline commands.go]: It might be useful if doMetrics automatically set `to` to `now` when only `--from` is specified?

File: commands.go [hunk 2]
```diff
@@ -147,6 +149,25 @@ var commandThrow = cli.Command{
 	},
 }
 
+var commandMetrics = cli.Command{
+	Name:      "metrics",
+	Usage:     "Fetch metric values",
+	ArgsUsage: "[--host | -h <hostId>] [--service | -s <service>] stdin --from int --to int",
```
Comment (astj) [inline commands.go]: It looks like `commandMetrics` does not take `stdin` as input, but does `--name | -n <metricName>` .

**Comment by edangelion** [comment]
Thank you for reviews!

**Comment by astj** [comment]
Sorry, I've forgotten to merge! :bow:

**Pull request merged**